raspberry/estimators/altitude_Kalman_filter.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+from matplotlib import pyplot as plt
+
+class KalmanFilter:
+    def __init__(self, sampling_frequency=10, x_tilde_0=np.array([[0,]*4]).T, P_0=np.array([[1, 1, 1, 1],]*4), cache_values=False) -> None:
+        self.__fs = sampling_frequency
+        self.__Ts = 1/self.__fs
+
+        # Normalized Throttle reference, range [0, 1]
+        self.__Tt = 0.
+        # System dynamics
+        self.__At = self.__update_At()
+        self.__C = np.array([[1, 0, 0, 0]])
+
+        # Process noise
+        self.__Q = np.array([
```

Comment:
Why are you taking `Q` to be this matrix? This is not positive definite.

Reply:
That was a mistake. I am correcting it.

Reply:
Taking `Identity 4x4` as `Q`.